assets/src/scripts/monitoring-location/store/hydrograph-data.js

```diff
@@ -97,7 +97,7 @@ const retrieveIVData = function(siteno, dataKind, {parameterCode, period, startT
         const isCalculatedTemperatureCode = isCalculatedTemperature(parameterCode);
 
         return fetchTimeSeries({
-            sites: [siteno],
+            siteno: siteno,
```

Comment:
All of the fetch functions now take a single siteno and a single parameter code (which can be null). The code for this page only ever deals with a single site and we are either fetching a single parameter code or all of them. Having this mix of sometimes passing in a single siteno or sometimes an array of sitenos (and similarly for the parameter codes) is what caused the issue that I found when displaying groundwater levels when using custom date range (we were passing in a single siteno rather than an array). Also standardizing on siteno and parameterCode as the variable names

Reply:
Wonderful!